tox.ini

```diff
@@ -1,16 +1,15 @@
 [tox]
 envlist =
     py{35,36,37}-django22-drf{310,311,master},
```

Comment:
it should be possible to combine this line and the next two into one with something like the following (not tested):

```
py{35,36,37,38}-django{22,30}-drf{312,master}
```

Reply:
python 3.5 is EOL

Reply:
> python 3.5 is EOL

Django 2.2 supports python 3.5.

Reply:
> it should be possible to combine this line and the next two into one with something like the following (not tested):
> 
> ```
> py{35,36,37,38}-django{22,30}-drf{312,master}
> ```

I believe py35 is not supported by django 3, hence the separation there but I can combine a couple of other lines. Testing now.

Reply:
fair point Django 3 indeed only supports 3.6+